build-support/scripts/consul-module-versions-in-consul.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+
+readonly SCRIPT_NAME="$(basename ${BASH_SOURCE[0]})"
+readonly SCRIPT_DIR="$(dirname "${BASH_SOURCE[0]}")"
+readonly SOURCE_DIR="$(dirname "$(dirname "${SCRIPT_DIR}")")"
+readonly FN_DIR="$(dirname "${SCRIPT_DIR}")/functions"
+
+source "${SCRIPT_DIR}/functions.sh"
+
+function usage {
+cat <<-EOF
+Usage: ${SCRIPT_NAME}  [<options ...>]
+
+Description:
+
+   This script is just a convenience around discover what the Consul
+   version would be if you were to build it. 
+
+Options:                       
+   -s | --source     DIR         Path to source to build.
+                                 Defaults to "${SOURCE_DIR}"
+                                 
+   -r | --release                Include the release in the version
+   
+   -g | --git                    Take git variables into account
+   
+   -h | --help                   Print this help text.
```

Comment:
This usage looks like copypasta - might need an update and flag removal?

Reply:
corrected in 2ae80130911529a8d7c7258027d690a00f720198